viral/basic_plots.py

```
@@ -0,0 +1,766 @@
+    y_mid = y_start + 0.5 * bar_dff
+    ax.text(
+        x_text_side,
+        y_mid,
+        dff_label,
+        ha="right",
+        va="center",
+        fontsize=9,
+        color="black",
+    )
+
+    plt.savefig(SERVER_PATH / "viral_plots" / "example_traces" / f"example_traces.png")
```

Ruff (0.14.4)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)